.github/workflows/ci-unit-broker-broker-gp1.yaml

```diff
@@ -40,39 +42,11 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - name: Cache local Maven repository
-        uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-
-
-      - name: Check if this pull request only changes documentation
-        id:   docs
-        uses: apache/pulsar-test-infra/diff-only@master
-        with:
-          args: site2 .github deployment .asf.yaml .ci ct.yaml
-
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
-        if: steps.docs.outputs.changed_only == 'no'
-        with:
-          java-version: 1.8
-
-      - name: Set up Maven
-        uses: apache/pulsar-test-infra/setup-maven@master
-        if: steps.docs.outputs.changed_only == 'no'
-        with:
-          maven-version: 3.6.1
-
-      - name: run unit tests install by skip tests
-        if: steps.docs.outputs.changed_only == 'no'
-        run: mvn clean install -DskipTests -pl pulsar-broker-auth-sasl -am
-
-      - name: run unit tests pulsar auth sasl
-        if: steps.docs.outputs.changed_only == 'no'
-        run: mvn test -DfailIfNoTests=false -pl pulsar-broker-auth-sasl
+      - name: run unit tests for pulsar-broker
+        run: |
+          mvn -B -q -ntp install -pl pulsar-broker -am -DskipTests
```

Comment:
This would be fetching snapshot dependencies for other modules and it will fail to compile whenever a commit is also touching more than just `pulsar-broker`. Rather why not just do install with `-Pcore-modules`